finger-cursor.ts

```diff
@@ -11,14 +12,18 @@ import {Component} from '@wonderlandengine/api';
  */
 export class FingerCursor extends Component {
     static TypeName = 'finger-cursor';
-    static Properties = {};
 
-    init() {
-        this.lastTarget = null;
-    }
+    lastTarget?: CursorTarget | null = null;
```

Comment:
```suggestion
    lastTarget: CursorTarget | null = null;
```

It's always defined :)